.circleci/config.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       - checkout
       - restore_cache:
           key: nix-store
+      - run: nix-env -iA nixpkgs.git
```

Comment:
Do we want to still try to `nix-build` as usual to see f this is also compatible outside nix flakes?

Reply:
Yup, this was just because CI was complaining about `git` not being present.